src/SFA.DAS.AssessorService.Application.Api/Controllers/ContactQueryController.cs

```diff
@@ -79,7 +79,7 @@ public async Task<IActionResult> GetAllContactsForOrganisation(string endPointAs
             return Ok(contacts);
         }
 
-        [HttpGet("user/{userName}", Name = "SearchContactByUserName")]
+        [HttpGet("username/{userName}", Name = "SearchContactByUserName")]
```

Comment:
This is required changed as it conflicted with [HttpGet("user/{id}", Name = "GetContactById")]